### ISSUE TITLE
Too broad suppression of `unused-argument` warnings

### ISSUE THREAD

**Issue by psrb** [PyCQA/pylint-django#249]
`pylint-django` currently suppresses the `unused-argument` warning when a function/method contains an argument called `request`. The reason to ignore these warnings for an argument named `request` was discussed in #155 and implemented with commit https://github.com/PyCQA/pylint-django/commit/e4c8fc70af33b227371c6e0e80b0dccf0ad157fa.

Unfortunately the warning is also suppressed for all other function arguments. Therefore every other unused argument of the function will not be reported. Unused arguments can be an indicator for bugs and these warnings should only be explicitly silenced.

### Example

``` python
def user_detail(request, user_id):
    # nothing is done with user_id
    return JsonResponse({'username': 'steve'})
```

Pylint should issue  a warning that `user_id` is unused. But because the function has an argument named `request` no warning is issued.

**Comment by fechnert** [comment]
We've encountered the same issue recently. Pylint was not able to detect that further arguments were unused.

It would be awesome to see the linked PR merged :+1:
